client/ayon_core/hosts/houdini/plugins/create/create_redshift_rop.py

```diff
@@ -54,22 +55,33 @@ def create(self, subset_name, instance_data, pre_create_data):
 
         # Set the linked rop to the Redshift ROP
         ipr_rop.parm("linked_rop").set(instance_node.path())
-
         ext = pre_create_data.get("image_format")
-        filepath = "{renders_dir}{subset_name}/{subset_name}.{fmt}".format(
-            renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
-            subset_name=subset_name,
-            fmt="${aov}.$F4.{ext}".format(aov="AOV", ext=ext)
-        )
+        multi_layered_mode = pre_create_data.get("multi_layered_mode")
 
         ext_format_index = {"exr": 0, "tif": 1, "jpg": 2, "png": 3}
+        multilayer_mode_index = {"No Multi-Layered EXR File": "1",
+                                 "Full Multi-Layered EXR File": "2" }
+
+        if multilayer_mode_index[multi_layered_mode] == "1": 
+            filepath = "{renders_dir}{subset_name}/{subset_name}.{fmt}".format(
+                renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
+                subset_name=subset_name,
+                fmt="${aov}.$F4.{ext}".format(aov="AOV", ext=ext)
+            )
+
+        elif multilayer_mode_index[multi_layered_mode] == "2":
+            filepath = "{renders_dir}{subset_name}/{subset_name}.$F4.{ext}".format(
+                renders_dir=hou.text.expandString("$HIP/pyblish/renders/"),
+                subset_name=subset_name,
+                ext=ext
+            )
 
         parms = {
             # Render frame range
             "trange": 1,
             # Redshift ROP settings
             "RS_outputFileNamePrefix": filepath,
-            "RS_outputMultilayerMode": "1",  # no multi-layered exr
+            "RS_outputMultilayerMode": multilayer_mode_index[multi_layered_mode],
```

Comment:
If it is multilayer just for exr, why dont we just do it when the users use exr as image format? 
```
    parms = {
        # Render frame range
        "trange": 1,
        # Redshift ROP settings
        "RS_outputFileNamePrefix": filepath,
        "RS_outputBeautyAOVSuffix": "beauty",
        "RS_outputFileFormat": ext_format_index[ext],
        "RS_aovMultipart":
    }
    if ext == "exr":
        parms["RS_outputMultilayerMode"] = multilayer_mode_index[multi_layered_mode]
```